src/main/java/dinkplugin/notifiers/ClueNotifier.java

```diff
@@ -84,6 +85,16 @@ public void onWidgetLoaded(WidgetLoaded event) {
         }
     }
 
+    public void onTick() {
+        // Track how many ticks occur where we only have partial clue data
+        if (clueCount == null != clueItems.isEmpty()) // XOR
+            badTicks++;
+
+        // Clear data if over 8 ticks pass with only partial parsing
+        if (badTicks > 8)
```

Comment:
Since some people like to open many caskets in succession, should this be less than 8? less than 4? (i'm not sure what's the max possible delay btwn widget & chat message)